metadata/io.jsonwebtoken/jjwt-gson/0.11.5/serialization-config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think we can remove the empty metadata files and their entries from `index.json`

Reply:
- Done.